tasks.py

```diff
@@ -40,7 +42,22 @@ def build_images(ctx, component):
             'DOCKER_HOST not set, if you get an error you might be missing something like "minikube start"'
         )
 
-    devops.tasks.build_images(ctx, components)
+    with build_images_context(components, dry_run):
+        devops.tasks.build_images(ctx, components, dry_run)
+
+
+@contextmanager
+def build_images_context(components, dry_run):
+    """
+    Context manager for building images.
+
+    To be extended as needed, e.g. copy files to be visible for Dockerfile
+    during the build and clean up them afterwards.
+
+    :param list components: The components to be built.
+    :param bool dry_run: True if it's a dry run.
+    """
+    yield
```

Comment:
would be nice to have an example structure like

```python
# Setup
try:
  yield
finally:
  # Teardown
```

Reply:
probably needs to have a `pass` in the `finally`

Reply:
Indeed a good idea. Updated accordingly.